gms/impl/src/main/java/com/linkedin/metadata/resources/entity/EntityResource.java

```diff
@@ -198,6 +219,30 @@
     return RestliUtils.toTask(() -> new StringArray(_searchService.getBrowsePaths(urnToEntityName(urn), urn)));
   }
 
+  /*
+  Used to enable writes in GMS after data migration is complete
+   */
+  @Action(name = "delete")
+  @Nonnull
+  public Task<RollbackResponse> deleteEntity(@ActionParam(PARAM_URN) @Nonnull String urnStr) throws URISyntaxException {
+    Urn urn = Urn.createFromString(urnStr);
+
+    return RestliUtils.toTask(() -> {
+      RollbackResponse response = new RollbackResponse();
+
+      EbeanEntityService.RollbackRunResult result = _entityService.rollbackUrn(urn);
+
+      List<AspectRowSummary> deletedRows = result.getRowsRolledBack();
+
+      response.setAspectsAffected(deletedRows.size());
+      response.setEntitiesAffected(deletedRows.stream().filter(row -> row.isKeyAspect()).count());
+      response.setAspectRowSummaries(
+          new AspectRowSummaryArray(deletedRows.subList(0, Math.min(100, deletedRows.size()))));
```

Comment:
Can this have duplicates?

Reply:
Nope- this only covers aspects that are of the most recent version

Reply:
so deleting an urn is really a rollback of latest version of each aspect?